app/src/main/res/layout/fragment_home_old.xml

```diff
@@ -293,20 +293,23 @@
                         android:layout_width="match_parent"
                         android:layout_height="match_parent"
                         android:gravity="center"
-                        android:padding="@dimen/default_padding"
+
                         android:clickable="true"
                         android:id="@+id/ll_beneficiaries"
                         android:foreground="?selectableItemBackground"
                         android:orientation="vertical">
                         <ImageView
                             android:layout_width="56dp"
                             android:layout_height="56dp"
+                            android:layout_marginLeft="16sp"
+                            android:layout_marginRight="16sp"
                             app:srcCompat="@drawable/ic_beneficiaries_48px"/>
                         <TextView
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
                             android:text="@string/beneficiaries"
                             android:gravity="center"
+                            android:padding="0dp"
```

Comment:
I don't think `padding` is needed, default padding of TextView is 0dp only. Check it and make changes accordingly here.